launch.go

```diff
@@ -23,7 +24,11 @@ type scmPath struct {
 
 var mkdirAll = os.MkdirAll
 var stat = os.Stat
+var matchString = regexp.MatchString
+var checkIfPr = checkIfPR
```

Comment:
Sort of questionable. Ideally you would be able to push data into the function that calls this one and just let it do its thing.